exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 var roomMaxCapacity = 25;
-var enoughSpaceInRoom; // finish this statement
+var enoughSpaceInRoom = roomMaxCapacity > moreStudentsThanMentors; // finish this statement
```

Comment:
updated to: var enoughSpaceInRoom = roomMaxCapacity >= (studentCount + mentorCount);